mlir/include/mlir/Dialect/OpenMP/OpenMPClauseOperands.h

```diff
@@ -198,6 +199,7 @@ struct SimdlenClauseOps {
 
 struct TaskReductionClauseOps {
   llvm::SmallVector<Value> taskReductionVars;
+  llvm::SmallVector<bool> taskReductionVarsByRef;
```

Comment:
I personally prefer this name "template" to what's done for in-reduction and reduction. Would you agree about making the change and renaming `InReductionClauseOps::inReductionVarsByRef` and `ReductionClauseOps::reductionVarsByRef`?